features/user/reset_password.feature

```diff
@@ -1,10 +1,18 @@
 @reset-password @notify @skip-staging
 Feature: Reset user password
 
-Scenario: Request password reset
+Background:
   Given I have a buyer user
   And I wait 2 seconds to ensure the reset token is created after the user
-  And I am on the /user/reset-password page
+
+Scenario: User has forgotten their password and requests a password reset
+  When I am on the homepage
```

Comment:
Shouldn't this also be a `Given`? (does it make any difference?)

Reply:
Probably (no it doesn't make any difference)